README.md

```diff
@@ -1,5 +1,5 @@
-# Data Product Hub Services Java SDK Version 0.0.4
-Java client library to interact with various [Data Product Hub Services]().
+# Data Product Exchange Java SDK Version 0.0.4
+Java client library to interact with various [Data Product Exchange Services]().
```

Comment:
```suggestion
Java client library to interact with various Data Product Exchange Service APIs.
```
Let's be as consistent as possible with the other SDKs.

Reply:
Updated as suggested